src/cases/containers/roles-and-access-container/roles-and-access-container.component.spec.ts

```diff
@@ -7,7 +7,7 @@ import { ExuiCommonLibModule } from '@hmcts/rpx-xui-common-lib';
 import { provideMockStore } from '@ngrx/store/testing';
 import { of } from 'rxjs/internal/observable/of';
 import { AllocateARoleLinkComponent, RoleAccessSectionComponent } from '../../components';
-import { CASEROLES } from '../../../../api/workAllocation2/constants/roles.mock.data';
+import { CASEROLES } from '../../../../api/workAllocation/constants/roles.mock.data';
```

Comment:
Should not use api folder - we should replicate mock on src if we want to use it